### DISCUSSION TITLE
Heroku is stopping to offer free tiers

### DISCUSSION THREAD

**Discussion by habi** [monicahq/monica#6282] [General]
Heroku has started it's next chapter and will stop offering free plans soon: https://blog.heroku.com/next-chapter
This is completely understandable, but an issue for small, personal instances of Monica set up for testing and trial and getting users in.

I've managed to move my instance I set up on Heroku to fly.io with https://fly.io/launch/heroku very easy, maybe it would be good to change the 'install' button in the README to point to fly.io for trying out Monica.

BTW: I'm planning to self-host Monica on a server at cyon.ch, but didn't really get through the whole convoluted process yet.

**Comment by mrpmohiburrahman** [comment]
Hi @habi, I also wanted to host it on fly.io.
Can you please give me a hint how to do that?
I wanted to directly deploy it on fly.io.

**Reply by habi** [reply]
I moved from Heroku to fly.iOS with their importer, and didn't set it up from scratch. Sorry, can't help.

**Reply by mrpmohiburrahman** [reply]
ok, no worries.

**Comment by mrpmohiburrahman** [comment]
Hi, @habi , me again!
So I'm trying to deploy it with heroku but facing problem "Unsupported cipher ..."
Can you please take a look [https://github.com/monicahq/monica/discussions/6450](https://github.com/monicahq/monica/discussions/6450)
